SCClassLibrary/Platform/Platform.sc

```diff
@@ -91,9 +91,12 @@ Platform {
 		}
 	}
 
-	loadStartupFiles { this.startupFiles.do{|afile|
-		afile = afile.standardizePath;
-		if(File.exists(afile), {afile.load})
+	loadStartupFiles {
```

Comment:
> if project = true, will it still load startup files relative to the config file?

No, currently just with  `project=true` it will load the startup files from the default paths as it was done in master branch. To turn that off you have to enable `excludeDefaultPaths`. In most situations I think `excludeDefaultPaths` would be turned on for a project, but I didn't want to remove the possibility of still loading from default paths on a project, so I kept the two flags separate. 

For projects, I would set `excludeDefaultPaths = true` and use `Startup.add` for startup code, via a class `initClass` method like [this](https://github.com/miguel-negrao/TestSCProject/blob/master/Classes/startup/ProjectStartup.sc). Usually I put such a class in a separate folder in the project folder, and then I can include or exclude via the lang config file.